tests/imagegpt/test_modeling_imagegpt.py

```diff
@@ -171,6 +171,12 @@ def get_config(
             reorder_and_upcast_attn=reorder_and_upcast_attn,
         )
 
+    def get_pipeline_config(self):
+        config = self.get_config()
+        config.vocab_size = 513
```

Comment:
This is necessary because the feature extractor will not respect `vocab_size` (it takes actual pixel values) so we need to use a large enough vocab and sequence to fit real world images used in those tests.

This modified only the pipeline tests so we can keep the original model tests strictly similar.